src/makeCalculator.js

```diff
@@ -4,7 +4,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return 'Cannot divide by zero!';
+      }
```

Comment:
[CODE KNOWLEDGE] - The `divide` method should not return a string when the division by zero occurs. Instead, it should handle the error internally, possibly by not performing the operation. The method should be consistent with the others and should return the calculator object to maintain chainability.